manuals/1.0/ja/tutorial_rest.md

```
@@ -0,0 +1,694 @@
+   - 同じ構造が常に同じ形で表現される
+   - クライアントの実装が容易になる
+
+3. ドキュメントとしての役割
+   - システムの全体像の把握
+   - 情報構造の共通理解
+
+## コレオグラフィー：状態遷移の定義
+
+コレオグラフィーでは、操作の種類に応じて状態遷移を定義します。ALPSでは、操作は以下の種類に分かれます：
+
+| 操作         | メソッド        | 	HTTP メソッド説明     |
```

Markdownlint

Column: 30
Hard tabs

(MD010, no-hard-tabs)